tests/utils.py

```diff
@@ -715,5 +715,10 @@ def query_memory(n=2):
     TEST_NDARRAYS = TEST_NDARRAYS + (gpu_tensor,)  # type: ignore
 
 
+TEST_DEVICES = [[torch.device("cpu")]]
+if torch.cuda.is_available():
+    TEST_DEVICES.append([torch.device("cuda")])
```

Comment:
yes in our multigpu testing environment cuda:0 is mostly more precise. we can make it configurable (by id) and robust, such as:
```py
def get_test_devices(id=["0"]):
   ...
   if torch.cuda.device_count() < len(id)
   ...
```
but this could be done in the future PRs if we encounter actual memory issues.